tidb-binlog/deploy-tidb-binlog.md

```diff
@@ -370,9 +370,12 @@ The following part shows how to use Pump and Drainer based on the nodes above.
         # zookeeper-addrs = "127.0.0.1:2181"
         # kafka-addrs = "127.0.0.1:9092"
         # kafka-version = "0.8.2.0"
+        # The maximum number of messages (number of binlogs) in a broker request. If it is left blank or a value smaller than 0 is configured, the default value 1024 is used.
```

Comment:
not sure
```suggestion
        # The maximum number of messages (number of binlogs) in a broker request. If it is left blank or a value less than 0 is configured, the default value 1024 will be used.
```

Reply:
> not sure

Thanks for your suggestion. 
1. Both 'smaller than' and 'less than' are correct here.
2. In technical documents, we seldom use the future tense. In 'if' clauses, if the meaning is clear, we also prefer the present tense.